xattrvi

```diff
@@ -117,11 +118,12 @@ with open(tmpfile.name, 'w') as f:
 		f.write("# attributes in other namespaces:\n\n")
 		f.write("\n#".join([ "{0:10}   : {1}".format(key, value) for key, value in otherxattrs]))
 
+# code for editor quirks
```

Comment:
It's no longer code for editor quirks, we should drop that comment.

As far as I have sees through the code, this is the only remaining thing before merge. :)